### PR TITLE
feat: add and log a very rough entropy approximation

### PR DESCRIPTION
**Pull request by SahilJain314**

### CODE REVIEW

File: nemo_rl/algorithms/loss_functions.py [hunk 1]
```diff
@@ -163,17 +163,25 @@ def __call__(
...
         else:
-            actor_importance_weights = torch.ones_like(prev_logprobs)
+            importance_weights_to_use = torch.ones_like(prev_logprobs)
         actor_loss = masked_mean(
-            actor_importance_weights * torch.max(loss1, loss2), mask
+            importance_weights_to_use * torch.max(loss1, loss2), mask
         )
+
+        # Approximating entropy as E_{s ~ \pi_{gen}(s)}[-(\pi_{curr}/\pi_{gen})log(\pi_{curr}(s))]
+        with torch.no_grad():
```
Comment (parthchadha) [inline nemo_rl/algorithms/loss_functions.py:253]: @SahilJain314 can you add an explanation in our docs regarding this metric and what should be the expected behavior when things work/break?

Comment (SahilJain314) [reply nemo_rl/algorithms/loss_functions.py:253]: done: https://github.com/NVIDIA/NeMo-RL/pull/342/files#diff-cf14e4e37c4f917f01c9f810692ab4337281350a678660176edb8f7593b9e7a9R193
